packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -189,6 +189,27 @@ class ImagePickerPlugin extends ImagePickerPlatform {
     return Future.wait<XFile>(resized);
   }
 
+  /// Injects a file input, and returns a list of XFile media that the user selected locally.
+  @override
+  Future<List<XFile>> getMedia({
+    required MediaOptions options,
+  }) async {
+    final List<XFile> images = await getFiles(
+      accept: '$_kAcceptImageMimeType,$_kAcceptVideoMimeType',
+      multiple: options.allowMultiple,
+    );
+    final Iterable<Future<XFile>> resized = images.map(
+      (XFile image) => _imageResizer.resizeImageIfNeeded(
```

Comment:
The ImageResizer doesn't support "video/*" formats, I think the `resizeImageIfNeeded` method needs to be tweaked a little bit more so it only acts in mime-types that start with "image/". The scape hatch that needs a little bit of love lives here:

* https://github.com/flutter/packages/blob/0906c5f96e543731728322e2771e8323f6b19986/packages/image_picker/image_picker_for_web/lib/src/image_resizer.dart#L20

(Other than this small detail, looks good!)